app/javascript/dashboard/components/widgets/conversation/ConversationCard.vue

```diff
@@ -330,7 +339,7 @@ export default {
       router.push({ path: frontendURL(path) });
     },
     onCardHover() {
-      this.hovered = !this.hideThumbnail;
+      // this.hovered = !this.hideThumbnail;
```

Comment:
Is this intentional?

Reply:
Not intentional, I did it for testing. I have reverted the changes now.